app/src/main/java/com/example/tasky/agenda/presentation/itemdetail/viewmodel/EventDetailViewModel.kt

```diff
@@ -120,6 +123,71 @@ class EventDetailViewModel @Inject constructor(
                     showDatePicker = false
                 )
             }
+
+            EventDetailEvent.AddAttendeeClicked ->
+                state = state.copy(
+                    attendeeDialogState = AttendeeDialogState(
+                        show = true
+                    )
+                )
+
+            EventDetailEvent.AddAttendeeCloseClicked ->
+                state = state.copy(
+                    attendeeDialogState = state.attendeeDialogState.copy(
+                        show = false
+                    )
+                )
+
+            EventDetailEvent.AddButtonClicked -> {
+                state = state.copy(
+                    attendeeDialogState = state.attendeeDialogState.copy(
+                        isLoading = true
+                    )
+                )
+
+                viewModelScope.launch {
+                    val attendee = repository.getValidUser(state.attendeeDialogState.email)
+
+                    state = if (attendee != null) {
+                        state.copy(
+                            attendeeDialogState = state.attendeeDialogState.copy(
+                                isLoading = false,
+                                showSuccess = true
+                            ),
+                            event = state.event.copy(
+                                attendees = state.event.attendees + Attendee(
+                                    email = attendee.email,
+                                    fullName = attendee.fullName,
+                                    userId = attendee.userId,
+                                    eventId = state.eventId,
+                                    isGoing = true,
+                                    remindAt = state.event.remindAt
```

Comment:
Mmm why are you setting the remindat of the attendee?
Technically speaking, the current user should never know about the other participant´s remind at time